modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase

```diff
@@ -122,16 +122,33 @@ definition {
 	test GuestConfigurationCanPersistInBrowserLocalStorage {
 		property portal.acceptance = "true";
 
-		//task ("Given When Then") {
+		task ("Given guest user") {
+			SignOut.signOut();
+		}
 
-		// // Given guest user
-		// // When toggle underline to OFF state
-		// // And open a new browser tab
-		// // And open accessibility menu
-		// // Then menu option is set to OFF state
+		task ("When toggle underline to OFF state") {
+			AccessibilityMenu.openAccessibilityMenu();
 
-		//}
+			AssertNotChecked.assertNotCheckedNotVisible(
+				key_toggleSwitchLabel = "Underlined Links",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+		}
+
+		task ("And open a new browser tab") {
+			var portalURL = PropsUtil.get("portal.url");
```

Comment:
I think we need to add another step to toggle an option on, then toggle off, then check on the other tab, what do you think?

Reply:
@john-co

Reply:
The test scenarios here assumed ON state was the default state. You can also just change all OFF state to ON state. This will likely apply to all other test scenarios.